api/src/chat/services/bot.service.ts

```diff
@@ -70,14 +71,13 @@ export class BotService {
     );
     // Process message : Replace tokens with context data and then send the message
     const recipient = event.getSender();
-    const envelope: StdOutgoingEnvelope =
-      await this.blockService.processMessage(
-        block,
-        context,
-        recipient?.context,
-        fallback,
-        conservationId,
-      );
+    const envelope = (await this.blockService.processMessage(
+      block,
+      context,
+      recipient?.context as SubscriberContext,
+      fallback,
+      conservationId,
+    )) as StdOutgoingEnvelope;
```

Comment:
```suggestion
    ));
```
To check if this is still needed.